.eslintrc.js

```diff
@@ -9,7 +9,9 @@ module.exports = {
       rules: {
         'react-perf/jsx-no-new-array-as-prop': 'off',
         'react-perf/jsx-no-new-function-as-prop': 'off',
-        'react-perf/jsx-no-new-object-as-prop': 'off'
+        'react-perf/jsx-no-new-object-as-prop': 'off',
+        'space-infix-ops': 'off',
+        '@typescript-eslint/space-infix-ops': ['error', { 'int32Hint': false }]
```

Comment:
I prefer for "styling nits" to be warnings instead of errors, and reserving errors for things that will actually break. At least for me it's a smoother development experience.